.gitignore

```diff
@@ -2,3 +2,4 @@
 bin/
 .DS_Store
 _output
+.openshift-tests-extension
```

Comment:
I am wondering why the file showed up.

Reply:
I added this line otherwise we need to update the file every time when we add/remove tests

Reply:
Removed this line